source-browser.html

```diff
@@ -166,28 +191,57 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
               ),
             );
         });
-
+          
         $('#topics, #loading').toggleClass('display-none');
       }
-
+        
       function refreshTopics() {
         localStorage.clear();
         showTopics();
       }
-
+        
       function showSearchInput() {
         $('#search')
           .removeClass('display-none')
           .focus();
       }
-
+        
+        
+        var array = ['languages', 'trunk', 'staging', 'nursery', 'incubator', 'tools'];
+        array.forEach(function(header) {
+          $("#navbar").append("<li><a href=#"+header+" id=li"+header+">"+header+"</a></li>");
+          $("#inline-block m0").attr("id", name);
```

Comment:
What element is this adding to? Why is there an element with this misleading id that sounds like a class?